app/scripts/controllers/transactions/lib/util.js

```diff
@@ -13,24 +11,26 @@ const normalizers = {
 }
 
 /**
-  normalizes txParams
-  @param {Object} txParams
-  @returns {Object} - normalized txParams
+ * Normalizes the given txParams
+ * @param {Object} txParams - the tx params
+ * @param {boolean} lowerCase - whether to return the addresses lower cased
+ * @returns {Object} the normalized tx params
  */
-export function normalizeTxParams (txParams, LowerCase) {
+export function normalizeTxParams (txParams, lowerCase) {
   // apply only keys in the normalizers
   const normalizedTxParams = {}
   for (const key in normalizers) {
     if (txParams[key]) {
-      normalizedTxParams[key] = normalizers[key](txParams[key], LowerCase)
+      normalizedTxParams[key] = normalizers[key](txParams[key], lowerCase)
     }
   }
   return normalizedTxParams
 }
 
 /**
-  validates txParams
-  @param {Object} txParams
+ * Validates the given tx parameters
+ * @param {Object} txParams the tx params
```

Comment:
Nit: There's a dash missing here
```suggestion
 * @param {Object} txParams - the tx params
```